src/utils/validation/basic_data/addEmployeeValidation.ts

```diff
@@ -72,7 +72,7 @@ export const userValidationSchema = Yup.object<YupShape<FlatCristinPerson>>({
     !nvi?.verifiedAt.id
```

Comment:
Skjønner ikke hvorfor NVI skal ha noen betydning her.. Burde hvertfall ikke påvirke validering av person når man skal opprette ny ved innlogging 🤔

Reply:
Ser dette kommer fra PR som la til støtte for å opprette person uten f.nr.: https://github.com/BIBSYSDEV/NVA-Frontend/pull/4745.
Men ble ikke mye klokere på det egentlig. Får ta en snakk med @iBuzza når han er tilbake fra kurs om hva dette er for noe, og om det burde tas vekk når vi skal opprette ny person